packages/runtime/test/src/interpreter/interpreter.ts

```diff
@@ -125,4 +129,53 @@ describe("Interpreter", function () {
 			);
 		});
 	});
+	describe("ed25519verify", () => {
```

Comment:
I think you should put this test to the `opcode-list.ts` file because this one should be only test interpreter